lib/glimr_api_client/api.rb

```diff
@@ -10,15 +10,20 @@ module Api
         'https://glimr-api.taxtribunals.dsd.io/Live_API/api/tdsapi'
     )
 
+    # TODO: extract error handling method
```

Comment:
Agreed.  

Worth noting that there are two different categories of error handling going on here 😕. The `handle_response_errors` method deals with api server errors while the rescue deals with network errors.